### PR TITLE
Drip or Drown; Premier style update. And some other clothing related stuff.

### PR DESCRIPTION
**Pull request by cdb-is-not-good**
First and foremost, it's been like four years - No one has come with a better set of equipment for the Premier in terms of aesthetics and quality.

Replaces much of the Premiers old/mismatched green shit with newer eris captain sprites, if we want to re-color this a bit that's fine but either way we /desperately/ need to replace these ancient sprites.

Premier additionally now actually starts with their coat, and gets a pair of dress shoes instead of the old scuffed brown shoes.

Mind, the original hat/coat/uniform are still available as alternatives if for SOME reason you want to dress up like a christmas tree. I did not add an alt for the space armor/helmet due to them A. not matching and B. being old sprites. I guess if someone REALLY wants I'll port the /tg/ carapace armor/helmets more up to date sprites as an alt.

stewards 'hazard' rig has had its sprite changed to match th combat rig(newer, sleeker sprite - more in line with the old retainer suit they used to have.) stats remain unchanged. There's probably more steward/premier gear that will need to be replaced but since NO ONE plays these jobs I guess we'll see.

Ports the funny cyberpunk jacket from eris(in the loadout.)

ports the eris preacher robes icons, but doesn't code them in quite yet. I couldn't be fucked, there's SO many.

Actually adds the formal IH uniform in as well as doing some tweaks to the icon so that WO + spec can also get a formal uniform with their normal rank pips

ports the eris syndie berets.

Adds to the greyson loot pool an armored void using sprites from Près de l'oiseau#2625 over on the Eris discord. 

Replaces some more,  old sprites. Miner suit is replaced as pictured as well as the industrial RIG - sprites by Près once more.

Fixes the WOs helmet lacking the verb to turn on its lamp despite having the lamp coded in.

Edit; Fixes the syndie beret north facing sprite. 

Edit 2: actually codes in the neotheo sprites as alts for the Primes coat/hat. see pics, 9 alt hats 5 alt robes

As always, a big thanks to the talented spriters over at Eris

image
image
image

## About The Pull Request
<details>
<summary>
	About The Pull Request
</summary>
<hr>

	
<hr>
</details>

## Changelog
:cl:

/:cl:

### CODE REVIEW

**Comment by Trilbyspaceclone** [comment]
Hey hey if its not to much trouble make sure to credit the prs or people that made the sprites or what not!
Worst case if its a hassle
//taken from eris
//Taken from Tg
next to the sprites or above the item

File: code/modules/clothing/spacesuits/syndi.dm [hunk 1]
```diff
@@ -66,3 +66,38 @@
 		rad = 25
 	)
 	siemens_coefficient = 0.6
+
+//Upgraded version of the above, found rarely in greyson loot.
```
Comment (Trilbyspaceclone) [inline code/modules/clothing/spacesuits/syndi.dm:70]: were does this spawn to be rarely in GP loot?

Comment (cdb-is-not-good) [reply code/modules/clothing/spacesuits/syndi.dm:70]: It was added to the list of greyson clothing spawns with weight 1 - compared to the weight 8 of the regular space suit

**Comment by cdb-is-not-good** [comment]
> Hey hey if its not to much trouble make sure to credit the prs or people that made the sprites or what not! Worst case if its a hassle //taken from eris //Taken from Tg next to the sprites or above the item

Ah sure, you've got me there I'll go through and do that in a moment here.

**Pull request merged**